Content designer adding/leaving process for 2nd Line

kevindew opened the pull request:
Trello: https://trello.com/c/m1f54bYg/183-define-process-for-content-designers-to-work-on-smart-answers

This outlines the reasons why a content designer may have access to
GitHub and the processes that someone on 2nd Line should follow to
grant/remove their access.

Note to self:

- [ ] After merge update [wiki](https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/515440654/Adding+and+removing+users+to+and+from+the+GOV.UK+content+designers+GitHub+team)

tijmenb commented on source/manual/content-designer-github-access.html.md:
```diff
@@ -0,0 +1,60 @@
+   details for each designer: their name, GitHub username and role in
+   GOV.UK - [example][user-example]
+1. Raise this as a pull request
+1. Once PR is merged email gds-github-owners@digital.cabinet-office.gov.uk
```
Can we change this to govuk-senior-tech-members@digital.cabinet-office.gov.uk? The GDS GitHub owners mailing list goes out to a lot more people, who won't have context. I wouldn't want to receive emails for access requests in other parts of GDS.

bevanloon replied:
@tijmenb 👍  fixed in 212475f

kevindew commented:
@tijmenb I've got those changes in there for you - and reworked the text slightly so it's less 2nd line focussed (I expect mostly that people on 2nd line will be asked to do this - but do agree that this doesn't seem 2nd line specific)

Also with the change to email address, the gds-github-owners was consistent with our leaver process - I've now updated the leaver ticket to also use the govuk-senior-tech-members address https://trello.com/c/IQIV54Pc/72-leaver-template-v20 FYI

tijmenb commented:
@kevindew thank you!
